plugins/src/main/java/opengrok/auth/plugin/ldap/LdapFacade.java

```diff
@@ -307,8 +307,18 @@ public SearchControls getSearchControls() {
         return res;
     }
 
-    private String getSearchDescription(String dn, String filter, String[] attributes) {
-        return "DN: " + dn + " , filter: " + filter + " , attributes: " + String.join(",", attributes);
+    // available for testing
+    static String getSearchDescription(String dn, String filter, String[] attributes) {
+        StringBuilder builder = new StringBuilder();
+        builder.append("DN: ");
+        builder.append(dn);
+        builder.append(" , filter: ");
```

Comment:
is the extra space before the comma necessary?

Reply:
removed